### ISSUE TITLE
BreadCrumb: Improve accessibility

### ISSUE THREAD

**Issue by Rawi01** [primefaces/primefaces#4928]
## 1) Environment
- PrimeFaces version: **latest**
- Does it work on the newest released PrimeFaces version? Version? **No**
- Does it work on the newest sources in GitHub? (Build by source -> https://github.com/primefaces/primefaces/wiki/Building-From-Source) **No**
- Application server + version: **-**
- Affected browsers: **-**

## 2) Expected behavior

According to the ARIA best practices site (https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html) a breadcrumb should be:
- rendered as ordered list
- use no additional aria roles
- hide seperators
- set `aria-current="page"` for the last link

## 3) Actual behavior

The breadcrumbs are an unordered list using `menu` aria roles.

---
I fixed that for my project using a custom renderer and can provide a pull request if you think that this should be changed.

**Comment by melloware** [comment]
I would submit a PR.

**Comment by Rawi01** [comment]
The fix changes the tag from `ul` to `ol`. This requires theme changes. 
It seems like the themes are generated, does it make sense to change these files instead of the used template?

**Comment by tandraschko** [comment]
Thats Up to @mertsincan

**Comment by melloware** [comment]
@jepsar has provided a nice PR for this and made it backwards compatible.

**Comment by jepsar** [comment]
I added a draft PR with an example of how to style the accessible bread crumb: https://github.com/primefaces/primefaces-showcase/pull/198
